python/test/Test_NDItk_MultigroupTable.py

```diff
@@ -228,12 +439,59 @@ def verify_chunk( self, chunk ) :
                                  xs = ReactionCrossSections(
                                           xs = [ CrossSection( 2, 0., [ 10., 20., 30., 40., 50., 60., 70. ] ),
                                                  CrossSection( 16, 1.1234567, [ 1., 2., 3., 4., 5., 6., 7. ] ) ] ),
+                                 scattering = ScatteringMatrix(
+                                     [ LegendreMoment( 0, [ 1, 0, 0, 0, 0, 0, 0,
```

Comment:
Could you make just one or two of these numbers have decimals?

Reply:
I can do that.